.github/workflows/release.yml

```diff
@@ -31,7 +31,6 @@ jobs:
     steps:
       - uses: actions/checkout@v2
         with: 
-          fetch-depth: 0
```

Comment:
I was suggesting that we NEED `fetch-depth: 0` for both checkouts, not to remove it from the first one. Without it, you get only the last sha commit and can't switch to any branch as none exist.

Reply:
you're right, it must be added for both